src/components/reusable/tabs/tabs.ts

```diff
@@ -29,6 +30,16 @@ export class Tabs extends LitElement {
   @property({ type: Boolean })
   vertical = false;
 
+  /** Enables tab content change on focus with keyboard navigation/assistive technologies. . */
+  @property({ type: Boolean })
+  autoFocusUpdate = false;
```

Comment:
Default behavior should remain as change on focus.

Reply:
Done